### PR TITLE
adding DLP Quickstart + redact image sample

### PR DESCRIPTION
**Pull request by jabubake**
+Formatting fixes using new checkstyle.

FYI @ace-n

### CODE REVIEW

File: dlp/src/main/java/com/example/dlp/Inspect.java [hunk 1]
```diff
@@ -275,18 +285,19 @@ private static void inspectDatastore(String projectId, String namespaceId, Strin
       // infoTypes = ['US_MALE_NAME', 'US_FEMALE_NAME'];
 
       // Get reference to the file to be inspected
-      PartitionId partitionId = PartitionId.newBuilder().setProjectId(projectId)
-          .setNamespaceId(namespaceId).build();
+      PartitionId partitionId =
+          PartitionId.newBuilder().setProjectId(projectId).setNamespaceId(namespaceId).build();
```
Comment (lesv) [inline dlp/src/main/java/com/example/dlp/Inspect.java]: Wow - that seems long - is all that really necessary, if not, perhaps a comment

File: dlp/src/main/java/com/example/dlp/Inspect.java [hunk 2]
```diff
@@ -336,61 +351,37 @@ public static void main(String[] args) throws Exception {
...
-        .required(false)
-        .build();
+    Option bucketNameOption = Option.builder("bucketName").hasArg(true).required(false).build();
     commandLineOptions.addOption(bucketNameOption);
 
-    Option gcsFileNameOption = Option.builder("fileName")
-        .hasArg(true)
-        .required(false)
-        .build();
+    Option gcsFileNameOption = Option.builder("fileName").hasArg(true).required(false).build();
```
Comment (lesv) [inline dlp/src/main/java/com/example/dlp/Inspect.java]: Why did we move from many lines to a single line?

Comment (jabubake) [inline dlp/src/main/java/com/example/dlp/Inspect.java]: I used the Google Java formatter on these files.

File: dlp/src/test/java/com/example/dlp/QuickStartIT.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+public class QuickStartIT {
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+    assertNotNull(System.getenv("GOOGLE_APPLICATION_CREDENTIALS"));
```
Comment (lesv) [inline dlp/src/test/java/com/example/dlp/QuickStartIT.java]: Do we really need this?

Comment (jabubake) [inline dlp/src/test/java/com/example/dlp/QuickStartIT.java]: Unfortunately yes, DLP requires service account credentials set as env var

File: dlp/src/main/java/com/example/dlp/Inspect.java (hunk 1 above)
Comment (jabubake) [inline dlp/src/main/java/com/example/dlp/Inspect.java]: Added comments, unfortunately yes : necessary

**Pull request merged**